mmv1/third_party/terraform/.teamcity/components/inputs/services_ga.kt

```diff
@@ -546,6 +546,11 @@ var ServicesListGa = mapOf(
         "displayName" to "Networkservices",
         "path" to "./google/services/networkservices"
     ),
+    "networkservicesplugins" to mapOf(
```

Comment:
Thanks, I need to discuss it with the product owner (next week). I suppose that he would prefer something like "service_extensions" instead of "network_services_plugins".
AFAIK the main reason of moving it to the new product were failing tests for other network_services resources.
Do you think that there is a chance to make them passing? What should change on networkservices API?

Reply:
We prefer `networkservices` product, which means we have to wait to finalize this work until WasmPlugin API is available as `v1`.